servicetalk-opentelemetry-http/src/main/java/io/servicetalk/opentelemetry/http/OpenTelemetryHttpServerFilter.java

```diff
@@ -64,15 +64,15 @@ public final class OpenTelemetryHttpServerFilter extends AbstractOpenTelemetryHt
     @Deprecated // FIXME: 0.43 - remove deprecated ctor
     @SuppressWarnings("DeprecatedIsStillUsed")
     public OpenTelemetryHttpServerFilter(final OpenTelemetry openTelemetry) {
```

Comment:
While you are here, could you please update `OpenTelemetryHttpRequesterFilterTest` to use new builder instead of this ctor?